components/automate-ui/src/app/pages/project/list/project-list.component.ts

```diff
@@ -77,16 +85,29 @@ export class ProjectListComponent implements OnInit, OnDestroy {
     this.store.pipe(
       select(createStatus),
       takeUntil(this.isDestroyed),
-      filter(state => {
-        return this.creatingProject && state === EntityStatus.loadingSuccess;
-      }))
+      filter(state => this.creatingProject && state === EntityStatus.loadingSuccess))
       .subscribe(() => {
         this.creatingProject = false;
+
+        const createTeamsWasChecked = this.createProjectForm.controls['addTeams'].value;
+        const projectID = this.createProjectForm.controls['id'].value;
+        const projectName = this.createProjectForm.controls['name'].value;
+
         this.closeCreateModal();
 
         // This is issued periodically from projects-filter.effects.ts; we do it now
         // so the user doesn't have to wait.
         this.store.dispatch(new LoadOptions());
+
+        if (createTeamsWasChecked) {
+          for (const [teamID, teamName] of Object.entries(this.teamsToCreate)) {
+            this.store.dispatch(new CreateTeam({
+              id: `${projectID}-${teamID}`,
+              name: `${projectName} ${teamName}`,
+              projects: [projectID]
```

Comment:
Is there a reason we cannot make the addition of teams work more similar to the addition of policies? Seems odd that the two return different types of notifications, as shown.

If that approach is not viable, though, a comment on the current implementation:
Would like to have the notifications include the team id, so we do not see the repetitive "Created a new team". That is, should say "Created team "p1-foo-project-owners", etc.

<img width="981" alt="image" src="https://user-images.githubusercontent.com/6817500/83074683-75475800-a027-11ea-9c7d-c99aa8e93d7e.png">